postinstall.js

```diff
@@ -16,3 +16,8 @@ if (!fs.existsSync(path.join(__dirname, ".cygwin"))) {
 console.log("Restoring hardlinks...");
 restoreLinks();
 console.log("Complete!");
```

Comment:
This message will be printed before restoreLinks completes.

Reply:
Good catch!